src/components/panels/TimePanel.vue

```diff
@@ -11,9 +11,13 @@
                 night: time.state == 'night',
                 warm: time.state == 'warm',
                 cold: time.state == 'cold',
+                fass: time.active == 'fass',
+                vome: time.state == 'vome',
```

Comment:
should use `===`

(unsure why eslint didn't complain)

if possible, please fix the others, too


also...

```suggestion
                fass: time.active === 'fass',
                vome: time.active === 'vome',
```